test/cpp/ext/otel/otel_plugin_test.cc

```diff
@@ -1353,6 +1353,23 @@ TEST_F(OpenTelemetryPluginNPCMetricsTest,
               HistogramResultEq(kSum, kMin, kMax, kCount))))));
 }
 
+TEST(OpenTelemetryPluginMetricsEnablingDisablingTest, TestEnableDisableAPIs) {
+  grpc::internal::OpenTelemetryPluginBuilderImpl builder;
+  builder.DisableAllMetrics();
+  EXPECT_TRUE(builder.TestOnlyEnabledMetrics().empty());
+
```

Comment:
Nit: Please remove unnecessary blank lines in tests.

Reply:
removed